packages/frontend-api/src/Resources/config/graphql-types/ModelType/Pricing/PricingSettingDecorator.types.yaml

```diff
@@ -10,3 +10,6 @@ PricingSettingDecorator:
             minimumFractionDigits:
                 type: "Int!"
                 description: "Minimum number of decimal places for the price on the current domain"
+            sellingPriceType:
+                type: "Int!"
+                description: "Type of selling price (1 - with VAT, 2 - without VAT)"
```

Comment:
could you create an enum API type for this, please? It is much better usable on the storefront then :pray: